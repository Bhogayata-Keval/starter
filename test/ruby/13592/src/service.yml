# Generated by Cloud66 Starter
---
services:
  custom_web:
    git_url: http://github.com/kamerblauwlicht/starter.git
    git_branch: master
    command: bundle exec unicorn_rails -c config/unicorn.rb -E _env:RAILS_ENV -D
    build_command: /bin/sh -c "RAILS_ENV=_env:RAILS_ENV bundle exec rake db:schema:load"
    deploy_command: /bin/sh -c "RAILS_ENV=_env:RAILS_ENV bundle exec rake db:migrate"
    build_root: test/ruby/13592/src/
    ports:
      - container: 8080
        http: 80
        https: 443
    env_vars:
      RAILS_ENV: production
      RACK_ENV: production
  worker:
    git_url: http://github.com/kamerblauwlicht/starter.git
    git_branch: master
    command: bundle exec rake jobs:work
    build_root: test/ruby/13592/src/
    env_vars:
      RAILS_ENV: production
      RACK_ENV: production
databases:
  - postgresql
  - redis
