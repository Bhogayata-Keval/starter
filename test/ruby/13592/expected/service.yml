# Generated by Cloud66 Starter
---
services:

  custom_web:
    git_url: git@github.com:cloud66/starter.git
    git_branch: master
    command: bundle exec unicorn_rails -c config/unicorn.rb -E _env:RAILS_ENV -D
    build_command: /bin/sh -c "RAILS_ENV=_env:RAILS_ENV bundle exec rake db:schema:load"
    deploy_command: /bin/sh -c "RAILS_ENV=_env:RAILS_ENV bundle exec rake db:migrate"
    build_root: test/ruby/13592/src/
    ports:
      - container: 8080
        http: 80
        https: 443
    env_vars:
      RAILS_ENV: production
      RACK_ENV: production

databases:
  - mysql